connection-service/new-admin-vm/Install-ConnectionServer.ps1

```diff
@@ -874,6 +891,23 @@ brokerProductVersion=1.0
 brokerIpaddress=$ipaddressString
 brokerLocale=en_US
 "@
+              
+                $isMfa = $using:enableRadiusMfa
+#                Write-Host "MFA setting is $isMfa"
+#stick in RADIUS MFA related attributes if RADIUS MFA is turned on
+                if($isMfa -eq "True") {
+                    $localRadiusHost = $using:radiusServerHost
```

Comment:
inside the string (below) you can just use $using:radiusServerHost embedded in the string and avoid the local's. (I didn't know that when I wrote a bunch of this code.)

Reply:
I didn't know that. Fixing it.